src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.Common.targets

```diff
@@ -8,6 +8,8 @@
 
     <PropertyGroup Condition="'$(RoslynAssembliesPath)' == ''">
       <RoslynAssembliesPath>$(RoslynTargetsPath)</RoslynAssembliesPath>
+      <RoslynAssembliesPath Condition="'$(MSBuildProjectExtension)' == '.csproj'">$([System.IO.Path]::GetDirectoryName($(CSharpCoreTargetsPath)))</RoslynAssembliesPath>
```

Comment:
should we use the `Language` property instead? What if the project is an fsharp project?

Reply:
> should we use the Language property instead? 

i decided against it because the sdk was using this MSBuildProjectExtension property to make this decision while setting this *CoreTargetsPath properties from RoslynTargetsPath.

> What if the project is an fsharp project

we dont overrride this property in the compilers package hence will use the default sdk version from RoslynAssembliesPath

Reply:
A ok, sounds good.